SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/SpatialRPCService.cpp

```diff
@@ -70,7 +72,7 @@ EPushRPCResult SpatialRPCService::PushRPC(const Worker_EntityId EntityId, const
 
 	if (RPCRingBufferUtils::ShouldQueueOverflowed(Type) && ClientServerRPCs.ContainsOverflowedRPC(EntityType))
 	{
-		if (EventTracer != nullptr)
+		if (EventTracer != nullptr && EventTracer->IsEnabled())
```

Comment:
&& EventTracer->IsEnabled()

Was this failing some tests or something?

Reply:
Nope! Just concerned about performance implications of this code in critical paths and we check IsEnabled inside `TraceEvent`. Seems like a much smaller cost to pay when ET is disabled